renovate-shared-config.json

```diff
@@ -31,6 +31,23 @@
     }
   ],
   "packageRules": [
+    {
+    // These files in these repos are synced from the bootc-dev/infra repository, which
+    // sends PRs to update them. Ignoring them here to avoid conflicting Renovate updates.
+      "matchRepositories": [
```

Comment:
This section will need to be updated anytime a new repo is added to the org.

How about adding a CI check which calls a script which both checks and can update this?